coalib/settings/Section.py

```diff
@@ -62,7 +62,7 @@ def __init__(self,
         self.contents = OrderedDict()
 
     def bear_dirs(self):
-        bear_dirs = path_list(self.get("bear_dirs", ""))
+        bear_dirs = glob_list(self.get("bear_dirs", ""))
```

Comment:
@sils1297 How do we do tests here? This basically proofs against glob characters interfering, but none of the CI (Appveyor, Travis, Circle) have these characters such that we can test if they work properly.